Makefile

```
@@ -0,0 +1,46 @@
+.PHONY: test test-unit test-integration test-coverage test-coverage-html build vet fmt lint clean help
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)